sdk/tests/iot/hub/test_az_iot_hub_client.c

```diff
@@ -335,7 +335,7 @@ static void test_az_iot_hub_client_get_user_name_with_model_id_small_buffer_firs
   assert_int_equal(
       az_iot_hub_client_init(&client, test_hub_hostname, test_device_id, &options), AZ_OK);
 
-  char mqtt_topic_buf[sizeof(test_correct_user_name) + 2];
+  char mqtt_topic_buf[sizeof(test_correct_user_name) - 1];
```

Comment:
Note: this change is due to the fact that the preview name is now shorter. This compares the user name without a model id (`test_correct_user_name`) vs with a model id (`test_correct_user_name_with_model_id`).

Reply:
I don't understand. How does that result in a change of 3 fewer chars?